satellite/console/service.go

```diff
@@ -141,6 +119,37 @@ func (s *Service) CreateUser(ctx context.Context, user CreateUser, tokenSecret R
 		return nil, errs.New(internalErrMsg)
 	}
 
+	// TODO: remove after vanguard release
+	var registrationToken *RegistrationToken
+	if user.PartnerID != "" {
```

Comment:
Could you add some comments about what is going on here ? Why the difference etc.